cumulusci/conftest.py

```diff
@@ -51,3 +53,22 @@ def _make_response(status):
         return MockHttpResponse(status)
 
     return _make_response
+
+
+@fixture(scope="session")
+def fallback_orgconfig(request):
```

Comment:
Is depending on the request here important for some reason?

Reply:
No, I've taken it out.

https://github.com/SFDO-Tooling/CumulusCI/pull/1816/files#diff-6fc3836ef3692ac53afe24241373e54dR59